### PR TITLE
cta: Update main action to book a demo

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
The journey for our customers is seldomly through the OSS edition of FlowFuse. While over the past year and a half we've had many installs it's for the business more important to get a demo on the books than start their FlowFuse adoption journey.

Screenshot 2023-08-21 at 15 29 40

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Review by zackwasli — APPROVED** [review]

File: src/index.njk
Comment (Yndira-FlowForge) [inline src/index.njk:125]: This image is for light backgrounds, let me add the right version before merging

**Review by iskerrett — APPROVED** [review]

File: src/index.njk
Comment (ZJvandeWeg) [reply src/index.njk]: Ok, feel free to merge right afterwards @Yndira-FlowForge

**Review by Yndira-FlowForge — APPROVED** [review]

**Pull request merged**
